controllers/profiles/reconciler_prod.go

```diff
@@ -19,6 +19,8 @@ import (
 	"fmt"
```

Comment:
maybe we cab create two mor packages
profiles/{prod,dev} and another package for common functions, or just leave it on profiles package

Reply:
We can do it but since this change is not related with the main goal of this activity, isn't it better to manage in a separate task?

Reply:
Sure thing, can you please open a new jira?

Reply:
Thanks.